### PR TITLE
List reports 

### PR DESCRIPTION
**Pull request by rileynwong**
Endpoint is reports/reports

### CODE REVIEW

**Comment by rileynwong** [comment]
r @kyle-rosenbluth

File: app/templates/reports/reports.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+            <div class="ui menu">
+                <div id="select-role" class="ui dropdown item">
+                    <div class="text">
+                        All incident reports
+                    </div>
+                    <i class="dropdown icon"></i>
+                    <div class="menu">
+                        <div class="item" data-value="">All incident reports
+</div>
```
Comment (kyle-rosenbluth) [inline app/templates/reports/reports.html:25]: This is weirdly indented

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.PHONY: clean
```
Comment (sandlerben) [inline Makefile:1]: Let's add this in a different branch/PR. It doesn't really have to do with `list-reports`

Also - can we add it as a git checkout hook?

File: app/models/incident_report.py [hunk 3]
```diff
@@ -22,6 +22,8 @@ class IncidentReport(db.Model):
                                backref='incident_report')
     date = db.Column(db.DateTime)  # hour the incident occurred
     duration = db.Column(db.Interval)  # like timedelta object
+    # agency = TODO: Hook this up with agency when Nancy finishes. TODO: set
```
Comment (sandlerben) [inline app/models/incident_report.py]: Make sure this branch has the latest changes from `agency-schema`. I am pretty sure I removed these comments.

File: app/reports/views.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+from flask import render_template
+from flask.ext.login import login_required
+from . import reports
+from ..models import IncidentReport, Agency
+
+
+@reports.route('/reports')
```
Comment (sandlerben) [inline app/reports/views.py:7]: I would make the route something different. How about `/reports/all`?

File: app/reports/views.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+@login_required
+def view_reports():
+    """View all idling incident reports."""
```
Comment (sandlerben) [inline app/reports/views.py:10]: Thank you for the comment :100: 

Consider adding more details about how you can only see certain reports depending on your `role`.

File: tests/test_agency_model.py [hunk 6]
```diff
@@ -43,4 +43,3 @@ def test_agency_with_incident_report_no_user(self):
         self.assertTrue(agency.is_official)
         self.assertFalse(agency.is_public)
         self.assertEqual(agency.incident_reports, [incident1, incident2])
```
Comment (sandlerben) [inline tests/test_agency_model.py:45]: Why did you change this file?

File: app/models/incident_report.py (hunk 3 above)
Comment (sandlerben) [inline app/models/incident_report.py]: I just tried that and it didn't fix it. I will cherry-pick these commits onto the `agency-schema` branch and re-push this.

**Comment by sandlerben** [comment]
@nancywong the git history was a little messed up so I made a new branch and cherry-picked the four commits where you made changes.

The problem was that you branched this off of your local `agency-schema` branch which was out of sync with the remote branch. Next time, make sure your local branch is in perfect sync before checking out.

I figured that the git history was off because the `tmp` commit was added again as part of this PR.

File: app/reports/views.py [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+    reports = IncidentReport.query.all()
```
Comment (sandlerben) [inline app/reports/views.py]: This shadows the name of the blueprint (see line 3)

**Comment by sandlerben** [comment]
Looks pretty good, but I still want to run it locally with some real reports.

We can leave this PR unmerged for a little bit while we add more features like filtering.
